straxen/plugins/peaklet_processing.py

```diff
@@ -186,6 +186,47 @@ def clip_peaklet_times(peaklets, start, end):
                 p['time'] = start
             if strax.endtime(p) > end:
                 p['length'] = (end - p['time']) // p['dt']
+                
+                
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_tpc_pmts_he',track=False,default=800
+        ),
+    strax.Option(
+        'channel_offset_he',track=False,default=500
```

Comment:
you can use https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L25 first argument. No need for an extra option.

Reply:
No you cannot. This is executed during the init, you do not have acces yet to straxen.whatevers. Also relative paths do not work (from ..contexts import whatever)

Reply:
Hmm I see, odd. Can you set the top to 752 than at least (to make utterly sure you are not doing any aqmon monitoring stuff) and give a description of the option

Reply:
Aqm channels are already split off